### PR TITLE
multi language error and bug 1993

### PR DESCRIPTION
**Pull request by GSuma21**

### CODE REVIEW

File: projects/lib-project/src/lib/dock/components/project-details/project-details.component.ts [hunk 1]
```diff
@@ -253,6 +266,10 @@ export class ProjectDetailsComponent implements OnDestroy, OnInit, AfterViewChec
           }
       })
   }
+
+  updateErrors() {
```
Comment (ramkumar-ns-dev) [inline projects/lib-project/src/lib/dock/components/project-details/project-details.component.ts]: why empty function ?

File: projects/lib-project/src/lib/dock/components/tasks/tasks.component.ts [hunk 2]
```diff
@@ -37,6 +37,7 @@ export class TasksComponent implements OnInit, OnDestroy {
   viewOnly: boolean = false;
   mode: any = "";
   commentPayload:any;
+  testingVariable:string = 'tasks[0]'
```
Comment (ramkumar-ns-dev) [inline projects/lib-project/src/lib/dock/components/tasks/tasks.component.ts]: can you remove this ?

File: projects/lib-project/src/lib/dock/components/tasks/tasks.component.ts [hunk 3]
```diff
@@ -55,6 +56,7 @@ export class TasksComponent implements OnInit, OnDestroy {
         this.taskFileTypes = this.tasksData?.fileType.options.map((item:any) => item.value)
       })
     )
+    console.log(this.libProjectService.projectData+this.testingVariable)
```
Comment (ramkumar-ns-dev) [inline projects/lib-project/src/lib/dock/components/tasks/tasks.component.ts]: remove this as well.

File: projects/lib-shared-modules/src/lib/services/http-provider.service.ts [hunk 4]
```diff
@@ -74,4 +77,14 @@ export class HttpProviderService {
       options.headers = options.headers.set('X-Requested-With', 'XMLHttpRequest');
       return options;
     }
+
+    openSnackBar(data:any) {
```
Comment (ramkumar-ns-dev) [inline projects/lib-shared-modules/src/lib/services/http-provider.service.ts]: please remove this function

**Review by ramkumar-pacewisdom — CHANGES_REQUESTED** [review]

**Review by ramkumar-pacewisdom — APPROVED** [review]

**Pull request merged**